lib/macho/headers.rb

```diff
@@ -412,38 +412,38 @@ module MachO
     MH_DYLIB_STUB => :dylib_stub,
     MH_DSYM => :dsym,
     MH_KEXT_BUNDLE => :kext_bundle,
-  }
+  }.freeze
 
   # association of mach header flag symbols to values
   # @api private
   MH_FLAGS = {
-    :MH_NOUNDEFS => 0x1,
-    :MH_INCRLINK => 0x2,
-    :MH_DYLDLINK => 0x4,
-    :MH_BINDATLOAD => 0x8,
-    :MH_PREBOUND => 0x10,
-    :MH_SPLIT_SEGS => 0x20,
-    :MH_LAZY_INIT => 0x40,
-    :MH_TWOLEVEL => 0x80,
-    :MH_FORCE_FLAT => 0x100,
-    :MH_NOMULTIDEFS => 0x200,
-    :MH_NOPREFIXBINDING => 0x400,
-    :MH_PREBINDABLE => 0x800,
-    :MH_ALLMODSBOUND => 0x1000,
-    :MH_SUBSECTIONS_VIA_SYMBOLS => 0x2000,
-    :MH_CANONICAL => 0x4000,
-    :MH_WEAK_DEFINES => 0x8000,
-    :MH_BINDS_TO_WEAK => 0x10000,
-    :MH_ALLOW_STACK_EXECUTION => 0x20000,
-    :MH_ROOT_SAFE => 0x40000,
-    :MH_SETUID_SAFE => 0x80000,
-    :MH_NO_REEXPORTED_DYLIBS => 0x100000,
-    :MH_PIE => 0x200000,
-    :MH_DEAD_STRIPPABLE_DYLIB => 0x400000,
-    :MH_HAS_TLV_DESCRIPTORS => 0x800000,
-    :MH_NO_HEAP_EXECUTION => 0x1000000,
-    :MH_APP_EXTENSION_SAFE => 0x02000000
-  }
+    MH_NOUNDEFS: 0x1,
+    MH_INCRLINK: 0x2,
+    MH_DYLDLINK: 0x4,
+    MH_BINDATLOAD: 0x8,
+    MH_PREBOUND: 0x10,
+    MH_SPLIT_SEGS: 0x20,
+    MH_LAZY_INIT: 0x40,
+    MH_TWOLEVEL: 0x80,
+    MH_FORCE_FLAT: 0x100,
+    MH_NOMULTIDEFS: 0x200,
+    MH_NOPREFIXBINDING: 0x400,
+    MH_PREBINDABLE: 0x800,
+    MH_ALLMODSBOUND: 0x1000,
+    MH_SUBSECTIONS_VIA_SYMBOLS: 0x2000,
+    MH_CANONICAL: 0x4000,
+    MH_WEAK_DEFINES: 0x8000,
+    MH_BINDS_TO_WEAK: 0x10000,
+    MH_ALLOW_STACK_EXECUTION: 0x20000,
+    MH_ROOT_SAFE: 0x40000,
+    MH_SETUID_SAFE: 0x80000,
+    MH_NO_REEXPORTED_DYLIBS: 0x100000,
+    MH_PIE: 0x200000,
+    MH_DEAD_STRIPPABLE_DYLIB: 0x400000,
+    MH_HAS_TLV_DESCRIPTORS: 0x800000,
+    MH_NO_HEAP_EXECUTION: 0x1000000,
+    MH_APP_EXTENSION_SAFE: 0x02000000
```

Comment:
I'm not happy to give up Ruby 1.8 compatibility just yet and this change makes restoring compatibility a lot harder. To that end, would you mind adding the following to the `.rubocop.yml` and reverting this and similar changes?

```yaml
# Ruby 1.8 compatibility
Style/DotPosition:
  EnforcedStyle: trailing
Style/HashSyntax:
  EnforcedStyle: hash_rockets
Style/TrailingCommaInArguments:
  EnforcedStyleForMultiline: no_comma
```